velox/exec/RowsStreamingWindowBuild.cpp

```diff
@@ -68,7 +82,16 @@ void RowsStreamingWindowBuild::addInput(RowVectorPtr input) {
     }
 
     if (previousRow_ != nullptr && inputRows_.size() >= numRowsPerOutput_) {
-      addPartitionInputs(false);
+      // No need to wait the peer group ready for rows frame.
```

Comment:
RowsStreamingWindowBuild is used with RANGE frames that are CURRENT ROW limited only. It is not applicable for k Range Frames. K Range frames would need k peers. It might be worth asserting that the range frame is not k bounded.